### ISSUE TITLE
MelonDS - "Could not find ROM" ???

### ISSUE THREAD

**Issue by kimdonga99** [rafaelvcaetano/melonDS-android#1358]
I tried to connect from Pegasus(frontend) to MelonDS, but it says"Could not find ROM". Is there a way?

It works fine in melonds, So I'm confused.

collection: NDS
extension: nds
launch: am start --user 0
-a android.intent.action.VIEW
-n me.magnum.melonds/.ui.emulator.EmulatorActivity
-e PATH "{file.path}"

I also specified the ROM directory in the application. I don't know why it's not working.

PLS Help me

![zWRJTtU](https://github.com/user-attachments/assets/c14199d2-b169-4fdc-ab92-9bda413ff0ef)
![20241118_225737](https://github.com/user-attachments/assets/24e2bc39-46f2-499f-8e0a-48db3ef17823)
![20241118_225818](https://github.com/user-attachments/assets/e5680bd1-9ba5-4761-bd5b-7c233025194f)

**Comment by kimdonga99** [comment]
I tested it with internal storage and it worked.
Is it not possible to use an SD card?
The nds is too big

**Comment by kimdonga99** [comment]
Not exactly, but maybe I can add permissions to the storage space in the app?
I think it will be solved if you give me official authority.

**Comment by rafaelvcaetano** [comment]
Having it on the internal storage or on an SD card shouldn't make any difference. But other users have reported this issue as well and I haven't found the cause yet.

**Comment by kimdonga99** [comment]
Any news? I'm sure this is right.

**Comment by rafaelvcaetano** [comment]
Folder permissions are no longer a thing on Android. They changed so that now, when we want the user to select a folder, we have to use their folder picker. When the user selects a folder, then the device assumes that the user gave permission to the app to access that folder. What you are seeing in that screenshot is a way to give access to everything, which Google won't accept unless a very good reason is provided when publishing an update to the Play Store.
As such, I'm still looking into this.

**Comment by 1ray0ng** [comment]
Facing the same problem with Beacon.
